docs/adrs/0001-aws-backup.md

```diff
@@ -0,0 +1,36 @@
+
+# Backups with AWS Backup
+
+## Status
+
+proposed
+
+### TODO
+
+ - [ ] Spike solution
+ - [ ] Spike AWS Config vs. Tag Policies
+ - [ ] Decide whether opt-out mechanism is 
+
+## Context
+
+A well-architected AWS setup includes backups for non-transient resources like RDS/Aurora, DynamoDB, EFS etc. so superwerker should protect users by enabling backups automatically.
+Since superwerker prefers the usage of native AWS services, AWS Backup is used.
+AWS Backup does not currently support backing up all resources in an AWS account. Either ARNs or Tags have to be speficied. A workaround has to be found.
```

Comment:
Typo. Semantically: we could think about implementing an inheritance mechanism for accounts and OUs (hierarchically) that initially only AWS Backup supported resource types. Drawback: this Lambda most likely needs either CT CAR support or depends on a shared infra pipeline - I'd think CAR support would be the easiest in the beginning (deployment is master/payer triggered anyways) and the integrity protection for the tag prefix could be easily deployed on the top level as well.